athenz-sia.env

```diff
@@ -1,8 +1,13 @@
+# Guidelines for managing the env file here:
+#   - If a default value is present, it must be provided with the URL where the default value is set
+#   - Follow the order of env variables as listed on https://github.com/AthenZ/k8s-athenz-sia/blob/main/pkg/config/config.go
+
```

Comment:
```suggestion
#######################################################################################
# Guidelines for managing the env file here:
#   - If a default value is present, it must be provided with the URL where the default value is set
#   - Follow the order of env variables as listed on https://github.com/AthenZ/k8s-athenz-sia/blob/main/pkg/config/config.go
#######################################################################################
```
No blank line without # please

Reply:
fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/43/commits/0234685a24ad480f71ba31f057151819ae91803e